lisp/ess-r-mode.el

```diff
@@ -2260,7 +2260,40 @@ state.")
         ess-help-sec-regex ess-help-r-sec-regex
         ess-help-sec-keys-alist ess-help-r-sec-keys-alist ; TODO: Still necessary?
         inferior-ess-help-command inferior-ess-r-help-command
-        inferior-ess-help-filetype nil))
+        inferior-ess-help-filetype nil)
+  (when ess-help-add-help-links
+    (ess-r-help-add-links)))
+
+(define-button-type 'ess-r-help-link
+  'follow-link t
+  'action (lambda (_) (ess-r-help-button-action)))
+
+(defun ess-r-help-button-action ()
+  "Display help for button at point."
+  (let ((text (get-text-property (point) 'ess-r-help-link-text)))
+    (ess-display-help-on-object text)))
+
+(defun ess-r-help-add-links ()
+  "Add links to the help buffer."
+  (save-excursion
+    ;; Search for fancy quotes only. If users have
+    ;; options(useFancyQuotes) set to something other than TRUE this
+    ;; probably won't work. If it's FALSE, R outputs ascii ', but
+    ;; searching through the whole buffer takes too long.
+    (while (re-search-forward
+            (rx (minimal-match (and "‘" (1+ not-newline) "’"))) nil t)
```

Comment:
Can we not rely on rx? I think the rx package is ridiculous idea. Everyone knows regex syntax from whatever language they are coming from. rx just makes people learn another emacs-only-much-longer "syntax" to achieve the same thing. I find it pretty cryptic.

Reply:
I like the rx syntax. It keeps me from having to remember how many times I have to escape things since that varies quite a bit across languages. But I'll take it out.